src/esp/assets/ResourceManager.cpp

```diff
@@ -82,6 +82,14 @@ constexpr char ResourceManager::DEFAULT_LIGHTING_KEY[];
 constexpr char ResourceManager::DEFAULT_MATERIAL_KEY[];
 constexpr char ResourceManager::WHITE_MATERIAL_KEY[];
 constexpr char ResourceManager::PER_VERTEX_OBJECT_ID_MATERIAL_KEY[];
+
+namespace {
+bool isRenderAssetGeneral(AssetType type) {
+  return type == AssetType::MP3D_MESH || type == AssetType::UNKNOWN ||
+         type == AssetType::SUNCG_OBJECT;
```

Comment:
I chatted with Kathryn Duvall from FB legal and she says this will be fine. What I'm trying to do here is not break the SUNCG loading codepath, with the big caveat that we can't test this loading path on SUNCG data or otherwise use SUNCG data. We should consider support for SUNCG data to be deprecated. I will add a runtime warning about this codepath being deprecated.

Reply:
Awesome! Thanks for following up on this!